x-pack/plugins/infra/public/components/waffle/waffle_group_by_controls.tsx

```diff
@@ -12,62 +12,169 @@ import {
   EuiFilterGroup,
   EuiPopover,
 } from '@elastic/eui';
+import { FormattedMessage, InjectedIntl, injectI18n } from '@kbn/i18n/react';
 import React from 'react';
 import { InfraNodeType, InfraPathInput, InfraPathType } from '../../../common/graphql/types';
 
 interface Props {
   nodeType: InfraNodeType;
   groupBy: InfraPathInput[];
   onChange: (groupBy: InfraPathInput[]) => void;
+  intl: InjectedIntl;
 }
 
-const OPTIONS = {
-  [InfraNodeType.pod]: [
-    { text: 'Namespace', type: InfraPathType.terms, field: 'kubernetes.namespace' },
-    { text: 'Node', type: InfraPathType.terms, field: 'kubernetes.node.name' },
-  ],
-  [InfraNodeType.container]: [
-    { text: 'Host', type: InfraPathType.terms, field: 'host.name' },
-    { text: 'Availability Zone', type: InfraPathType.terms, field: 'meta.cloud.availability_zone' },
-    { text: 'Machine Type', type: InfraPathType.terms, field: 'meta.cloud.machine_type' },
-    { text: 'Project ID', type: InfraPathType.terms, field: 'meta.cloud.project_id' },
-    { text: 'Provider', type: InfraPathType.terms, field: 'meta.cloud.provider' },
-  ],
-  [InfraNodeType.host]: [
-    { text: 'Availability Zone', type: InfraPathType.terms, field: 'meta.cloud.availability_zone' },
-    { text: 'Machine Type', type: InfraPathType.terms, field: 'meta.cloud.machine_type' },
-    { text: 'Project ID', type: InfraPathType.terms, field: 'meta.cloud.project_id' },
-    { text: 'Cloud Provider', type: InfraPathType.terms, field: 'meta.cloud.provider' },
-  ],
+let OPTIONS: { [P in InfraNodeType]: Array<{ text: string; type: InfraPathType; field: string }> };
```

Comment:
I don't think these options should be internationalized -- they directly map to the field names in the reported data, these field names will not be internationalized either. They are technical terms, and changing them will make the UI more confusing for the user.